docs/run/processor-types.mdx

```diff
@@ -12,6 +12,17 @@ Processor types are named for the general technology qualities that go into buil
 
 Illustration of segment H on a Falcon processor.
 
+
+## Heron
+
+![Heron processor icon](/images/run/processor-types/heron.svg)
+
+Quantum volume: 512
+
+At 133 qubits, Heron is an [Eagle](#eagle)-sized upgrade to [Egret](#egret) that requires pulling in substantial innovations in signal delivery that were previously deployed in [Osprey](#osprey). The signals required to enable the fast, high-fidelity two-qubit and single-qubit control are delivered with high-density flex cabling.
```

Comment:
IDK if we have any input here, but "requires pulling in" instead of "pulls in" seems weird to me

Reply:
I think I'll take that chance and make your suggestion :)